README.md

```diff
@@ -51,18 +44,12 @@ For external links, you might want to consider the Path Redirect or
 Global Redirect modules, which allow you to set forwarding either per item or
 across the site to your aliased URLs.
 
-URLs (not) Getting Replaced With Aliases:
-Please bear in mind that only URLs passed through Drupal's l() or url()
-functions will be replaced with their aliases during page output. If a module
-or your template contains hardcoded links, such as 'href="node/$node->nid"'
-those won't get replaced with their corresponding aliases. Use the
-Drupal API instead:
-
-* 'href="'. url("node/$node->nid") .'"' or
-* l("Your link title", "node/$node->nid")
-
-See http://api.drupal.org/api/HEAD/function/url and
-http://api.drupal.org/api/HEAD/function/l for more information.
+###URLs (not) Getting Replaced With Aliases:
+Please bear in mind that only URLs passed through Drupal's Drupal's URL and
```

Comment:
How about it now @Berdir? I removed the links to the API functions as there are many ways to generate links. We could add a documentation link when there is one.